common/src/main/resources/assets/minecraft_access/lang/en_us.json

```diff
@@ -153,6 +153,8 @@
   "minecraft_access.gui.read_crosshair_config_menu.button.speak_block_sides_button": "Speak Block Sides",
   "minecraft_access.gui.read_crosshair_config_menu.button.use_jade_button": "Use Jade",
   "minecraft_access.gui.read_crosshair_config_menu.button.use_jade_button.unavailable": "Use Jade: Unavailable",
+  "minecraft_access.gui.mouse_simulation.button.toggle_mac_mouse_fix_button.enabled": "Use MacOS mouse fix: Enabled",
+  "minecraft_access.gui.mouse_simulation.button.toggle_mac_mouse_fix_button.disabled": "Use MacOS mouse fix: Disabled",
```

Comment:
It's better to reuse the `"minecraft_access.gui.common.button.toggle_button.disabled": "%s: Disabled",` and enabled i18n text, and pass the translated settings name into this with `I18n.translate(key,translatedName)`
Yeah, I know there are some old buttons that have idependent i18n text...